.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore image of Dryville
+wss-wu-dryville.jpg
```

Comment:
Not required, but you will see that red "no entry" symbol if you don't have a new line at the end of the file. You can fix it by adding a new empty line (just hit enter), saving, committing, and pushing.

Reply:
👍 I'll make the change and create a new PR

Reply:
*Just for my records* - Didn't need to make a new PR, since this one was still open, and therefore my commit to edit the .gitignore file was added to this open PR, which Lindsay then approved and merged